cloud/wfm/src/result/UrlResult.ts

```diff
@@ -0,0 +1,18 @@
+// NOTE: browserify can provide an implementation for node's url module
+import * as nodeUrl from 'url';
+import {Result} from './index';
+
+/**
+ * Represents a {@link Result} that can be accessed via a URL
+ */
+export default class UrlResult implements Result {
```

Comment:
Same as above let's provide more accurate example that will inherit type from actual step. Steps always define specific result and they have full control over this.